tests/system/pages/assetsPage/WatchOnlyAssetsPage.ts

```diff
@@ -15,4 +17,11 @@ export class WatchOnlyAssetsPage extends BasePage<AssetsPageElements> {
 
     return new WalletModalWindow(this.page, new WalletModalElements(), this);
   }
-}
+
+  public async checkTransferButtonNotExists(): Promise<WatchOnlyAssetsPage> {
+    const sendButton = this.page.getByTestId(TEST_IDS.ASSETS.SEND_ARROW_ICON).first();
+    await expect(sendButton).toHaveCount(0);
```

Comment:
would be better to use toBeVisible() in order to have logic semantic

```suggestion
    await expect(sendButton).not.toBeVisible();
```